ENIGMAsystem/SHELL/Widget_Systems/Unix/dialogs.cpp

```diff
@@ -0,0 +1,474 @@
+#include "dialogs.h"
+#include "Widget_Systems/widgets_mandatory.h"
+#include "tinyfiledialogs.h"
+#include <sys/stat.h>
+#include <limits.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <string.h>
+#include <algorithm>
+#include <sstream>
+#include <vector>
+#include <string>
+
+using std::string;
+
+namespace enigma
+{
+    bool widget_system_initialize()
+    {
+        return true;
+    }
+}
+
+bool is_number(string input)
+{
+    if (input[0] == '0' || input[0] == '1' || input[0] == '2' || input[0] == '3' || input[0] == '4' ||
```

Comment:
if( input[0] >= '0' && input[0] <= '9' )

Reply:
Cool! I had no idea you could do that with strings! :D